src/com/xilinx/rapidwright/util/BlackboxPopulator.java

```diff
@@ -136,7 +157,7 @@ private static boolean clearTargetSiteInsts(Module mod, Site proposedAnchorSite,
      * @param net The net to check
      * @return True if the net is a clock net
      */
-    public static boolean isClockNet(Net net) {
+    private static boolean isClockNet(Net net) {
```

Comment:
Have you considered whether `Net.isClockNet()` is appropriate rather than rolling your own? If not, can you leave a comment as to why?

Reply:
These clock nets can be without their driver. Thus, need this method.